shared-components/src/components/checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,34 @@
+import { DetailedHTMLProps, forwardRef, InputHTMLAttributes } from "react";
+
+import "./checkbox.scss";
+
+export interface Checkbox extends InputHTMLAttributes<HTMLInputElement> {
+  className?: string;
+  lightBackground: boolean;
+  labelText: string;
+  id: string;
```

Comment:
Det finnes en funksjon for å lage ider, så kan det være optional fortsatt. Så bruker du den supplied eller den genererte.